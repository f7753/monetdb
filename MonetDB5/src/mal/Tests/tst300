# Exception identifiers can be introduced at any place

	myNewException:="new exception";
	raise myNewException;
	printf("should not come here\n");
#exception block;
catch	myNewException;
	printf("catched exception\n");
exit	myNewException;
