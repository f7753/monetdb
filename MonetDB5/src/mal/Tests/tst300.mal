# Exception identifiers can be introduced at any place

	raise myNewException:= "new exception";
	printf("should not come here\n");
#exception block;
catch	myNewException;
	printf("catched exception\n");
exit	myNewException;
