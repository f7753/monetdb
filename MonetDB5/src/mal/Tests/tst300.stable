# Monet Database Server V5.0
# Copyright (c) 1993-2004, CWI. All rights reserved.
# compiled for 32biti686-pc-linux-gnu/32bit.
# config:/ufs/mk/monet5/src/mal/Tests/tst.conf
# dbfarm:/ufs/mk/monet5/Linux/var/MonetDB
# dbname:demo
## Exception identifiers can be introduced at any place
#
#	myNewException:="new exception";
#	raise myNewException;
#	printf("should not come here\n");
##exception block;
#catch	myNewException;
#	printf("catched exception\n");
#exit	myNewException;
#
function Admin.main():int;	# 0  (main:int)
    myNewException := "new exception";	# 1  (myNewException:str)<-($2:str)
    raise myNewException;	# 2  (myNewException:str)
    io.printf("should not come here\n");	# 3 SYSprint_str ($3:int)<-($4:str)
catch myNewException;	# 4  (myNewException:str) jump 6
    io.printf("catched exception\n");	# 5 SYSprint_str ($5:int)<-($6:str)
exit myNewException;	# 6  (myNewException:str)
end main;	# 7  
catched exception
>