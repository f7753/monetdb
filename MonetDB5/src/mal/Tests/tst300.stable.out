stdout of test 'tst300` in directory 'src/mal` itself:
# Monet Database Server V5.0
# Copyright (c) 1993-2004, CWI. All rights reserved.
# compiled for 32biti686-pc-linux-gnu/32bit.
# config:/ufs/mk/monet5/src/mal/Tests/tst.conf
# dbfarm:/ufs/mk/monet5/Linux/var/MonetDB/dbfarm
# dbname:demo
# Visit http://monetdb.cwi.nl for further information.
## Exception identifiers can be introduced at any place
#
#	raise myNewException:= "new exception";
#	printf("should not come here\n");
##exception block;
#catch	myNewException;
#	printf("catched exception\n");
#exit	myNewException;
#
function Admin.main():int;	# 0  (main:int)
    raise myNewException := "new exception";	# 1  (myNewException:str)<-($2:str)
    io.printf("should not come here\n");	# 2 IOprint_str ($3:int)<-($4:str)
catch myNewException;	# 3  (myNewException:str) jump 5
    io.printf("catched exception\n");	# 4 IOprint_str ($5:int)<-($6:str)
exit myNewException;	# 5  (myNewException:str)
end main;	# 6  
catched exception
>